test/contrib/automotive/testsocket.uts

```diff
@@ -51,7 +51,11 @@ for i in range(1, 2000, 2):
     pks.append(txp)
     pks.append(rxp)
 
-t.stop(join=True)
+try:
+    t.stop(join=True)
+except Scapy_Exception:
```

Comment:
Could you explain why an exception is acceptable?

Reply:
Yeah.. thats a big mistake. Thanks for asking. This should never be necessary.